package.json

```diff
@@ -7,7 +7,10 @@
     "type": "git",
     "url": "git://github.com/mapbox/polyline.git"
   },
-  "dependencies": {},
+  "dependencies": { 
+    "turf-flip": "*",
+    "geojson-coords": "*"
```

Comment:
We should probably add a version number here for both `turf-flip` and `geojson-coords`.